sda-commons-server-kafka/src/test/java/org/sdase/commons/server/kafka/dropwizard/KafkaWithoutHealthCheckTestApplication.java

```diff
@@ -0,0 +1,38 @@
+package org.sdase.commons.server.kafka.dropwizard;
+
+import com.codahale.metrics.health.HealthCheckRegistry;
+import io.dropwizard.Application;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import org.sdase.commons.server.dropwizard.bundles.ConfigurationSubstitutionBundle;
+import org.sdase.commons.server.kafka.KafkaBundle;
+
+public class KafkaWithoutHealthCheckTestApplication extends Application<KafkaTestConfiguration> {
+
+  private KafkaBundle<KafkaTestConfiguration> bundle =
+      KafkaBundle.builder()
+          .withConfigurationProvider(KafkaTestConfiguration::getKafka)
+          .withoutHealthCheck()
+          .build();
+
+  private HealthCheckRegistry healthCheckRegistry;
+
+  @Override
+  public void initialize(Bootstrap<KafkaTestConfiguration> bootstrap) {
+    bootstrap.addBundle(ConfigurationSubstitutionBundle.builder().build());
```

Comment:
You shouldn't need this bundle, as you aren't using env substitution

Reply:
Hm, without it the deserialisation of the config.yml failes

Reply:
Yeah you are right. But removing the env variable in the config would break other things